src/test/resources/com/puppycrawl/tools/checkstyle/checks/indentation/indentation/InputIndentationAnnotationArrayInitOldStyle.java

```diff
@@ -0,0 +1,27 @@
+/* Config:                                                                  //indent:0 exp:0
+ * This test-input is intended to be checked using following configuration: //indent:1 exp:1
```

Comment:
make it:
```
/*
Indentation
basicOffset = 4 
....
```

it is new format based on GSOC 21 project

Reply:
Done